src/view/boot.ts

```diff
@@ -18,6 +18,7 @@ import {MdIconRegistry} from '@angular2-material/icon/icon';
 import {CoreWebService} from "../api/services/core-web-service";
 import {LoginService} from "../api/services/login-service";
 import {RoutingService} from "../api/services/routing-service";
+import {FormatDate} from "../api/services/format-date-service";
```

Comment:
I think it is ok meanwhile we make a research about how to keep the app state, until that part I am ok with statefull services.